### PR TITLE
Remove Beta language from HCP packer docs

### PR DESCRIPTION
**Pull request by laurapacilio**
This PR removes the language about HCP Packer being in beta, in preparation for the GA launch.

### CODE REVIEW

**Review by im2nguyen — APPROVED** [review]

**Comment by laurapacilio** [comment]
also link checker is flagging hashicorp support site: https://support.hashicorp.com/hc/en-us which is definitely working, so we can ignore.

**Review by olena-seva — APPROVED** [review]

File: website/content/docs/datasources/hcp/hcp-packer-image.mdx [hunk 1]
```diff
@@ -19,8 +19,9 @@ The `HCP Packer Image` Data Source retrieves information about an
 image from the HCP Packer registry. This information can be used to
 provide a source image to various Packer builders.
 
-~> **Note:** HCP Packer is under active development, and we are currently offering a [public beta version](https://portal.cloud.hashicorp.com) to collect feedback and continue improving the product.
-To get started, visit the [HCP Packer documentation](https://cloud.hashicorp.com/docs/packer) or try the [Get Started with HCP Packer](https://learn.hashicorp.com/collections/packer/hcp-get-started) collection on HashiCorp Learn.
+To get started with HCP Packer, refer to the [HCP Packer documentation](https://cloud.hashicorp.com/docs/packer) or try the [Get Started with HCP Packer](https://learn.hashicorp.com/collections/packer/hcp-get-started) collection on HashiCorp Learn.
+
+~> **Note:** You will receive an error if you try to reference metadata from a deactivated or deleted registry. An administrator can manually deactivate or delete a registry, and HCP Packer automatically deactivates registries with billing issues. Contact [HashiCorp Support](https://support.hashicorp.com/) with questions.
```
Comment (laurapacilio) [inline website/content/docs/datasources/hcp/hcp-packer-image.mdx:24]:
```suggestion
~> **Note:** You will trigger an error if you try to reference metadata from a deactivated or deleted registry. An administrator can manually deactivate or delete a registry, and HCP Packer automatically deactivates registries with billing issues. Contact [HashiCorp Support](https://support.hashicorp.com/) with questions.
```

**Review by nywilken — APPROVED** [review]
LGTM. Merge once out of GA I take it.

**Review by azr — APPROVED** [review]
Nice !!! LGTM ! 🎉

**Pull request merged**